clean.js

```diff
@@ -0,0 +1,9 @@
+// clean script
+
+// use shelljs for cross platform
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
Why do we need this?

Reply:
if I remove it I got [eslint] 'shelljs' should be listed in the project's dependencies. Run 'npm i -S shelljs' to add it (import/no-extraneous-dependencies)

Reply:
I listed "shelljs" in dev-dependencies (I do not want to list it in dependencies)